src/cpu/minor/stats.cc

```diff
@@ -79,6 +95,10 @@ MinorStats::MinorStats(BaseCPU *base_cpu)
         .init(base_cpu->numThreads, enums::Num_OpClass)
         .flags(statistics::total | statistics::pdf | statistics::dist);
     committedInstType.ysubnames(enums::OpClassStrings);
+
+    commitedControl
+        .init(base_cpu->numThreads, enums::ControlInst::NumControlInstTypes);
```

Comment:
Can you use `StaticInstFlags` instead?

Reply:
Yeah, sure, but these functions are predefined and return the StaticInstFlags I believe. Just so I understand gem5 better, why would we not use this function?

Reply:
I was suggesting this to avoid creating a new enum.

Reply:
Enum StaticInstFlags.Flags of build/RISCV/enums/StaticInstFlags.hh has 36 different flags. From my understanding, if we use this our statistics vector will be of length 36 (currently it is 7). Would it be better to use those flags and extend the statistics to track all 36 StaticInst types?

Reply:
So use .flags(statistics::dist) I believe, right?

Reply:
I don't think so. See https://www.gem5.org/documentation/general_docs/statistics/ you can use the `nozero` flag.

Reply:
Oh yeah you're right, I was reading the wrong comment in src/base/stats/info.hh lol

Reply:
Removed the old enum and string map and used StaticInstFlags. Commit [315ae16](https://github.com/darchr/gem5/pull/247/commits/315ae16cbfad64f56622f14cb9437ec2f958fbe7).